tools/setup_perf_cluster.sh

```diff
@@ -71,7 +89,6 @@ function ExecuteEval() {
 
 function create_cluster() {
   Execute gcloud container clusters create $CLUSTER_NAME $GCP_OPTS --machine-type=$MACHINE_TYPE --num-nodes=$NUM_NODES --no-enable-legacy-authorization
-  Execute gcloud container clusters get-credentials $CLUSTER_NAME $GCP_OPTS
```

Comment:
why is this gone?

Reply:
there is another function kubectl_setup that does it. This one is redundant.